CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -488,6 +494,9 @@ class CameraFragment : Fragment() {
             framesPerSecond = 1.0 / ((timestampFirst - timestampLast) /
                     frameTimestamps.size.coerceAtLeast(1).toDouble()) * 1000.0
 
+            // Simulate long time analyze
+            Thread.sleep(1000)
+
```

Comment:
There's no need for this in the code, although I think leaving a comment explaining what you are trying to illustrate would be helpful. I suggest:
```
// Analysis could take an arbitrarily long amount of time
// Since we are running in a different thread, it won't stall other use cases
```